layersvt/screenshot.cpp

```diff
@@ -278,20 +269,20 @@ static void init_screenshot(const VkInstanceCreateInfo *pCreateInfo, const VkAll
         vkuGetLayerSettingValue(layerSettingSet, kSettingKeyFormat, value);
 
         if (value == "UNORM") {
-            userColorSpaceFormat = UNORM;
+            userColorSpaceFormat = colorSpaceFormat::UNORM;
         } else if (value == "SRGB") {
-            userColorSpaceFormat = SRGB;
+            userColorSpaceFormat = colorSpaceFormat::SRGB;
         } else if (value == "SNORM") {
-            userColorSpaceFormat = SNORM;
+            userColorSpaceFormat = colorSpaceFormat::SNORM;
         } else if (value == "USCALED") {
-            userColorSpaceFormat = USCALED;
+            userColorSpaceFormat = colorSpaceFormat::USCALED;
         } else if (value == "SSCALED") {
-            userColorSpaceFormat = SSCALED;
+            userColorSpaceFormat = colorSpaceFormat::SSCALED;
         } else if (value == "UINT") {
-            userColorSpaceFormat = UINT;
+            userColorSpaceFormat = colorSpaceFormat::UINT;
         } else if (value == "SINT") {
-            userColorSpaceFormat = SINT;
-        } else if (value == "USE_SWAPCHAIN_COLORSPACE") {
+            userColorSpaceFormat = colorSpaceFormat::SINT;
+        } else if (value != "USE_SWAPCHAIN_COLORSPACE") {
```

Comment:
To be clear this is the fix. I just also updated the code to use an enum class for my benefit.